plugin/src/main/groovy/com/novoda/staticanalysis/internal/detekt/DetektConfigurator.groovy

```diff
@@ -11,15 +12,17 @@ import org.gradle.api.Task
 class DetektConfigurator implements Configurator {
 
     private static final String DETEKT_PLUGIN = 'io.gitlab.arturbosch.detekt'
-    private static final String LAST_COMPATIBLE_DETEKT_VERSION = '1.0.0.RC9.2'
```

Comment:
Yeah, I keep updating this whenever I fix new problems :) To be honest, the problems so far were all very weird and were crashing earlier. It might be the case that this was never shown. :shrug:

But anyways, it is nice to have.